Assets/Scripts/Buddy/BackBuddy.cs

```diff
@@ -4,7 +4,9 @@
 public class BackBuddy : MonoBehaviour 
 {
 	[Tooltip( "Which blendshapes should be copied from the fed buddy." )]
-	MinMaxI copyBlendShapeIndicesRange = new MinMaxI( 3, 12 );
+	[SerializeField] MinMaxI copyBlendShapeIndicesRange = new MinMaxI( 3, 12 );
+
+	public BuddyStats _hiddenBuddy; // The buddy that is currently on the back.
```

Comment:
Actually this value is set by another script, so it needs to be public. I'll remove the leading underscore.